examples/tomo/xray_trafo.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2014, 2015 The ODL development group
```

Comment:
For each modality there is one image (2D) or three ortho-slices (3D) for each forward and backward projection for each geometry ie parallel2d, parallel3d, fanflat, circular conebeam, helical conebeam for each implementation gpu or cpu. If we want to check if all of these work properly how would you reduce the number of images? The ortho-slices for the phantom we could maybe delete.